org.librehunt.Organizer.json

```diff
@@ -0,0 +1,64 @@
+{
+    "app-id" : "org.librehunt.Organizer",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.30",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "organizer",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=home",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro"
+    ],
+    "build-options" : {
```

Comment:
Those build-options are now used by default on GNOME Sdk 3.30 (which is based on Freedesktop Sdk 18.08)

Reply:
these?
```
    "build-options" : {
        "cflags" : "-O2 -g",
        "cxxflags" : "-O2 -g",
        "env" : {
            "V" : "1"
        }
    }
```

Reply:
Yes

Reply:
well, I fixed that now, so is this good to merge? :D